node/rpc-api/src/data_types.rs

```diff
@@ -83,7 +84,7 @@ pub struct BlockMessages {
 #[derive(Serialize, Deserialize)]
 #[serde(rename_all = "PascalCase")]
 pub struct MessageSendSpec {
-    #[serde(with = "bigint_ser::json")]
+    #[serde(with = "bigint_ser")]
```

Comment:
This doesn't look right. The json serializer isn't the same as the binary serializer.

Reply:
Good catch.